main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
```

Comment:
Consider adding error handling for the file reading operation. If the file is missing or contains invalid JSON, it could cause the script to fail.